.github/workflows/pr.yaml

```diff
@@ -61,6 +99,8 @@ jobs:
   test:
     name: Test with Node.js ${{ matrix.node-version }}
```

Comment:
this is a required test. Can we run the step but have it immediately exit 0 if there's nothing to do?

![image](https://github.com/user-attachments/assets/7e0f733f-d703-4a54-9bf7-00a550565f97)